Signal/src/view controllers/SignalAttachment.swift

```diff
@@ -97,7 +97,25 @@ class SignalAttachment: NSObject {
             assert(false)
             return nil
         }
-        return "\(error)"
```

Comment:
nit: I think it would be more conventional for our errors to adopt the https://developer.apple.com/reference/foundation/localizederror protocol.

e.g. http://stackoverflow.com/a/39176551/353178